cypress/integration/adminFlows/pages/landingPage.spec.js

```diff
@@ -4,23 +4,27 @@ describe('Set a landing page from the admin portal', () => {
     cy.fixture('users/adminUser.json').as('user');
     cy.get('@user').then((user) => {
       cy.loginAndVisit(user, '/admin/customization/config').then(() => {
+        cy.get('#new_settings_user_experience').as('userExperienceSectionForm');
         // Ensure Forem instance is private
         // NOTE: @citizen428 - We may need to find a better situation for this
         // long-term.
-        cy.findByTestId('authSectionForm').as('authSectionForm');
 
-        cy.get('@authSectionForm').findByText('Authentication').click();
-        cy.get('@authSectionForm')
-          .findByLabelText('Invite-only mode')
-          .should('not.be.checked')
-          .check();
+        cy.get('@userExperienceSectionForm')
+          .findByText('User Experience and Brand')
```

Comment:
The only `findByText` that I was unable to change was here -- I wasn't sure if there was a way to differentiate between all of the "Show Info" buttons on the page that wouldn't cause issues down the road, say if one of the "Show Info"s changed in placement (we could probably grab the specific "Show Info" button for "User Experience and Brand" by grabbing the element's specific index, but I wasn't sure if that was best practice). cc @nickytonline

Reply:
You can find a heading role, and it should be all good. I tested this locally, and the tests still pass.

```suggestion
          .findByRole('heading', { name: 'User Experience and Brand' })
```

Reply:
Awesome, TIL! Thank you so much for the pointer.